met-web/src/components/comments/admin/review/CommentReview.tsx

```diff
@@ -47,24 +54,24 @@ const CommentReview = () => {
     const handleSave = async () => {
         setIsSaving(true);
         try {
-            await ReviewComment({ comment_id: Number(commentId), status_id: review });
+            await ReviewComment({ submission_id: Number(submissionId), status_id: review });
             setIsSaving(false);
-            dispatch(openNotification({ severity: 'success', text: 'Comment successfully reviewed.' }));
-            navigate(`/surveys/${comment.survey_id}/comments`);
+            dispatch(openNotification({ severity: 'success', text: 'Comments successfully reviewed.' }));
+            navigate(`/surveys/${comments[0].survey_id}/comments`);
         } catch (error) {
-            dispatch(openNotification({ severity: 'error', text: 'Error occurred while sending comment review.' }));
+            dispatch(openNotification({ severity: 'error', text: 'Error occurred while sending comments review.' }));
             setIsSaving(false);
         }
     };
 
-    const { id, comment_status, reviewed_by, submission_date, review_date, text } = comment;
+    const { submission_id, comment_status, reviewed_by, submission_date, review_date } = comments[0];
```

Comment:
not exactly part of this task 
but may be we should write some comment about why we deal with comments[0] alone ;
explain the logic that comments are approved togother and 0 and n will have same verdict..

Reply:
comment added